src/main/java/com/kazuma/quizapplication/QuizJFrame.java

```diff
@@ -193,6 +214,8 @@ private void initComponents()
         selectAnswerComboBox = new javax.swing.JComboBox<>();
         answerChoiceTextField = new javax.swing.JTextField();
         userGraphicJLabel = new javax.swing.JLabel();
+        jPanel1 = new javax.swing.JPanel();
+        lblTime = new javax.swing.JLabel();
```

Comment:
Do not use Hungarian Notation Prefixes![clcik here](http://facweb.cs.depaul.edu/sjost/it330/documents/hungarian.htm)
This should be timerLabel not lblTime